### PR TITLE
Inject environment variables into Xcode schemes.

### PR DESCRIPTION
**Pull request by shatlykuber**

### CODE REVIEW

**Comment by nguyentruongtho** [comment]
Looks good to me.

File: src/com/facebook/buck/apple/project_generator/SchemeGenerator.java [hunk 1]
```diff
@@ -311,6 +335,13 @@ public static Element serializeTestAction(Document doc, XCScheme.TestAction test
       testableElem.appendChild(refElem);
     }
 
+    if (testAction.getEnvironmentVariables().isPresent()) {
+      testActionElem.setAttribute("shouldUseLaunchSchemeArgsEnv", "NO");
```
Comment (robbertvanginkel) [inline src/com/facebook/buck/apple/project_generator/SchemeGenerator.java]: is this yes by default?

Comment (shatlykuber) [inline src/com/facebook/buck/apple/project_generator/SchemeGenerator.java]: yes, it is set to yes in the beginning of function.

**Comment by robbertvanginkel** [comment]
Then I say all good.

**Pull request merged**